src/aviary/utils.py

```diff
@@ -22,6 +22,21 @@
     "temperature": 0,
 }
 
+LLM_EXTRACT_CONFIG = {
+    "prompt": (
+        "You are evaluating answers for a test which has fixed options. "
```

Comment:
Can you add some statement that focuses the LLM on the message history?

Otherwise, in `paper-qa`, we witnessed the LLM using its innate knowledge

Reply:
There is no message history here (?) Not sure what you mean?

Reply:
I guess this function is responsible for both (1) extracting a letter that (2) ensuring it matches a multiple choice option.

What we saw in `paper-qa` was in the case of an empty string answer, the LLM would pull on its innate knowledge and could select the correct multiple choice option.

So for this:

> Can you add some statement that focuses the LLM on the message history?

I guess what I should of said was can you add a statement that focuses the LLM on just the `proposed: str`, and tries to avoid pulling on any innate knowledge?

Reply:
Yea - I was smart and didn't put the question into these, so there's no way it could get confused and try to answer.

Reply:
we simultaneously posted

There is no question. So I don't see how it would be possible for it to attempt to answer. I don't know what else I could write to make it more clear in the prompt.

Reply:
Ahhh I see, very clever! I follow and you're right

Do you mind documenting that rationale somewhere in the code? Maybe a docstring in `extract_answer_llm`